Sources/Sonar/APIs/AppleRadarRouter.swift

```diff
@@ -59,7 +60,7 @@ enum AppleRadarRouter {
                     "configurationSplitValue": radar.configuration,
                     "workAroundText": "",
                     "descriptionText": radar.body,
-                    "problemAreaTypeCode": radar.area.map { String($0.appleIdentifier) } ?? "",
+                    "problemAreaTypeCode": (radar.area.map { String($0.appleIdentifier) } ?? "") as Any,
```

Comment:
In 8.2.1:

![image](https://cloud.githubusercontent.com/assets/283886/23835563/b635abd4-0726-11e7-9e73-8fab1f40af68.png)